contracts/deploy/mainnet/118_multichain_strategist.js

```diff
@@ -0,0 +1,85 @@
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "118_multichain_strategist",
+    forceDeploy: false,
+    //forceSkip: true,
+    reduceQueueTime: true,
+    deployerIsProposer: false,
+    proposalId:
+      "29029286887383246507190904901480289164471194993284788972189545670391461770154",
+  },
+  async () => {
+    const { multichainStrategistAddr } = await getNamedAccounts();
+
+    const cOETHVaultProxy = await ethers.getContract("OETHVaultProxy");
+    const cOETHVault = await ethers.getContractAt(
+      "IVault",
+      cOETHVaultProxy.address
+    );
+
+    const cOUSDVaultProxy = await ethers.getContract("VaultProxy");
+    const cOUSDVault = await ethers.getContractAt(
+      "IVault",
+      cOUSDVaultProxy.address
+    );
+
+    const cOETHBuybackProxy = await ethers.getContract("OETHBuybackProxy");
+    const cOETHBuyback = await ethers.getContractAt(
+      "OETHBuyback",
+      cOETHBuybackProxy.address
+    );
+
+    const cOUSDBuybackProxy = await ethers.getContract("BuybackProxy");
+    const cOUSDBuyback = await ethers.getContractAt(
+      "OUSDBuyback",
+      cOUSDBuybackProxy.address
+    );
+
+    const cOETHHarvesterSimple = await ethers.getContract(
+      "OETHHarvesterSimple"
+    );
+
+    // const cAuraWETHPriceFeed = await ethers.getContract("AuraWETHPriceFeed")
+
```

Comment:
Should we remove this? Or it is done on purpose to match with the fifth governance actions?

Reply:
Removed it, thanks